core/src/main/scala/cats/data/Validated.scala

```diff
@@ -602,3 +602,38 @@ private[data] trait ValidatedFunctions {
   final def condNel[A, B](test: Boolean, b: => B, a: => A): ValidatedNel[A, B] =
     if (test) validNel(b) else invalidNel(a)
 }
+
+private[data] trait ValidatedFunctionsBinCompat0 {
+
+
+  /**
+   * Converts a `B` to a `ValidatedNec[A, B]`.
+   *
+   * For example:
+   * {{{
+   * scala> Validated.validNec[IllegalArgumentException, String]("Hello world")
+   * res0: ValidatedNec[IllegalArgumentException, String] = Valid(Hello world)
+   * }}}
+   */
+  def validNec[A, B](b: B): ValidatedNec[A, B] = Validated.Valid(b)
```

Comment:
validated is already covariant. What about `ValidatedNec[Nothing, B]` here?

Reply:
So, why do we need it? In cases where type inference falls down? e.g. in folds the initial value?